go/private/context.bzl

```diff
@@ -884,6 +887,11 @@ go_config = rule(
             providers = [BuildSettingInfo],
         ),
         "amd64": attr.string(),
+        "pgoprofile": attr.label(
+            mandatory = True,
+            providers = ["files"],
```

Comment:
This is a legacy restriction that I don't think does anything. We can probably just drop it.